tchannel-example/src/main/java/com/uber/tchannel/thrift/KeyValueClient.java

```diff
@@ -104,6 +116,8 @@ public static String getValue(
             throw err;
         }
 
+        getResult.release();
```

Comment:
the client always has to remember release? is there anyway to do self-release (like finalize)?